site/docs/src/json/scim/scim-extended-error-response.json

```diff
@@ -0,0 +1,24 @@
+{
+  "detail": "Request failed, see errors for additional details.",
+  "io.fusionauth.errors": {
+    "generalErrors": [
+      {
+        "code": "[invalid]",
+        "message": "Your JSON was invalid"
+      }
+    ],
+    "fieldErrors": {
+      "user.username": [
+        {
+          "code": "[duplicate]user.username",
+          "message": "A User with username = [neo] already exists."
+        }
+      ]
+    }
+  },
+  "schemas": [
+    "io.fusionauth.errors",
```

Comment:
Isn't this `urn:ietf:params:scim:schemas:extension:fusionauth:2.0:Error`.

Reply:
done